src/test/java/com/google/roomies/servlets/ListingsServletTest.java

```diff
@@ -76,6 +88,53 @@ public void setUp() throws Exception {
 
     DatabaseFactory.setDatabaseForTest(db);
     setRequestParameters();
+    defineDatabaseMockBehavior();
```

Comment:
I don't think this method is terrible useful, since it mostly just makes the reader go another step to figure out what's needed for setup. You could put it as a second @Before method if you want to keep all those when()'s grouped together and separate

Reply:
Moved into the setup method since I think it'll be easier for the reader to see it all in one place